CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing Guidelines
+
+Thanks for interest in contribution to `colin`.
+
+The following is a set of guidelines for contributing to `colin`. These are not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## How to contribute to colin
```

Comment:
How do I contribute code to colin?

Reply:
This is not part of this PR. I would separate it. I have created only the section.

Reply:
I'm sorry but that doesn't make any sense to me: you are creating a file called contributing.md and that file doesn't have any tips how to actually contribute code?